### ISSUE TITLE
Interactive 

### ISSUE THREAD

**Issue by d9k** [gnat/css-scope-inline#7]
Very cool concept, I love it.

But for real-world use case need some interactive input element in demo. 

For example, autocomplete city field which turns red if you type in name of non-existent city.

**Comment by nullndvoid** [comment]
Use some JavaScript such as Surreal or AlpineJS to write some code which sets a css class e.g. error.

**Comment by gnat** [comment]
Yup as mentioned by @nullndvoid I would highly recommend https://github.com/gnat/surreal as it's the same concept as CSS Scope Inline, but for `<script>` tags, in order to do client side interactions.

No plans currently to add a `<form>` demo but I'll keep it on my mind for future widgets to be added to the demo.

Will convert to a discussion for now.
